Source/Core/Core/PowerPC/BreakPoints.cpp

```diff
@@ -203,11 +202,22 @@ MemChecks::TMemChecksStr MemChecks::GetStrings() const
   {
     std::ostringstream ss;
     ss.imbue(std::locale::classic());
-
-    ss << std::hex << mc.start_address << " " << mc.end_address << " " << (mc.is_enabled ? "n" : "")
-       << (mc.is_break_on_read ? "r" : "") << (mc.is_break_on_write ? "w" : "")
-       << (mc.log_on_hit ? "l" : "") << (mc.break_on_hit ? "b" : "");
-    mc_strings.push_back(ss.str());
+    ss << fmt::format("${:08x} ", mc.start_address);
+    ss << fmt::format("{:08x} ", mc.end_address);
+    if (mc.is_enabled)
+      ss << 'n';
+    if (mc.is_break_on_read)
+      ss << "r";
+    if (mc.is_break_on_write)
+      ss << "w";
+    if (mc.log_on_hit)
+      ss << "l";
+    if (mc.break_on_hit)
+      ss << "b";
```

Comment:
A little nitpick, `'n'` is using char literal which is inconsistent with the other lines using string literal. Feel free to switch the other lines to use char literals too.

Reply:
`ss << "c " << mc.condition->GetText();`

Should this use fmt format or is it fine?